### DISCUSSION TITLE
[BUG]:[API V3] createShopShippingProfileDestination You must provide either a destination_country_iso code or a destination_region

### DISCUSSION THREAD

**Discussion by MaxLBrick** [etsy/open-api#991] [Q&A] [answered]
Hi,
According to description of createShopShippingProfileDestination method it is possible to not to pass destination_country_iso code or a destination_region in the parameters.
>  If the request sets neither destination_country_iso nor destination_region, the default destination is "everywhere".
https://developer.etsy.com/documentation/reference/#operation/createShopShippingProfileDestination

We tried that but got 400 Response stating "You must provide either a destination_country_iso code or a destination_region".
Our request looks like:
```php
$api->createShopShippingProfileDestination([
    'params' => [
        'shop_id' => $shopId,
        'shipping_profile_id' => $shippingProfileId,
    ],
    'data' => [
        'primary_cost' => round($regular1ItemDelivery, 2),
        'secondary_cost' => round(max(0, min($regular1ItemDelivery, $regular2thItemDelivery)), 2),
        'min_delivery_days' => $minDeliveryDays,
        'max_delivery_days' => $maxDeliveryDays,
    ],
]); 
```

Is it something wrong with our request or is it a bug?

**Comment by Gareth064** [comment, marked as answer]
Try setting `destination_country_iso` to null in your request. Might act differently then.

**Reply by MaxLBrick** [reply]
Thanks, @Gareth064 !
